articles/partner-twilio-python-how-to-use-voice-sms.md

```diff
@@ -17,16 +17,18 @@ ms.date: 02/19/2015
 ms.author: MicrosoftHelp@twilio.com
 
 ---
-# How to Use Twilio for Voice and SMS Capabilities in PHP
+# How to Use Twilio for Voice and SMS Capabilities in Python
 This guide demonstrates how to perform common programming tasks with the Twilio API service on Azure. The scenarios covered include making a phone call and sending a Short Message Service (SMS) message. For more information on Twilio and using voice and SMS in your applications, see the [Next Steps](#NextSteps) section.
 
 ## <a id="WhatIs"></a>What is Twilio?
 Twilio is powering the future of business communications, enabling developers to embed voice, VoIP, and messaging into applications. They virtualize all infrastructure needed in a cloud-based, global environment, exposing it through the Twilio communications API platform. Applications are simple to build and scalable. Enjoy flexibility with pay-as-you go pricing, and benefit from cloud reliability.
 
-**Twilio Voice** allows your applications to make and receive phone calls. **Twilio SMS** enables your application to send and receive text messages. **Twilio Client** allows you to make VoIP calls from any phone, tablet, or browser and supports WebRTC.
+**Twilio Voice** allows your applications to make and receive phone calls.
+**Twilio SMS** enables your application to send and receive text messages.
+**Twilio Client** allows you to make VoIP calls from any phone, tablet, or browser and supports WebRTC.
 
 ## <a id="Pricing"></a>Twilio Pricing and Special Offers
-Azure customers receive a [special offer] $10 of Twilio Credit when you upgrade your Twilio Account. This Twilio Credit can be applied to any Twilio usage ($10 credit equivalent to sending as many as 1,000 SMS messages or receiving up to 1000 inbound Voice minutes, depending on the location of your phone number and message or call destination). Redeem this Twilio credit and get started at: [ahoy.twilio.com/azure].
+Azure customers receive a [special offer][special_offer] $10 of Twilio Credit when you upgrade your Twilio Account. This Twilio Credit can be applied to any Twilio usage ($10 credit equivalent to sending as many as 1,000 SMS messages or receiving up to 1000 inbound Voice minutes, depending on the location of your phone number and message or call destination). Redeem this Twilio credit and get started at: [http://ahoy.twilio.com/azure][special_offer].
```

Comment:
Perhaps this could be changed
`Redeem this Twilio credit and get started at: [http://ahoy.twilio.com/azure][special_offer].`
to:
`Redeem this [Twilio credit][special_offer] and get started.`